code/conlleval.pl

```diff
@@ -16,7 +16,7 @@
 #            with $boundary fields (default -X-).
 # url:       http://www.clips.uantwerpen.be/conll2000/chunking/
 # started:   1998-09-25
-# version:   2004-01-26
+# version:   2018-03-09
 # author:    Erik Tjong Kim Sang <erikt@uia.ua.ac.be>
```

Comment:
Maybe change author to "original author" to have this more clear. Also indent the "with modification" under "Erik" to have the modification more clear